Updated SAA Template

eddie-knight opened the pull request:
Adjusted the SAA template to close #223.

Marking this as a draft pending feedback from the community.

AdrianHammond commented:
@eddie-knight  - few comments on [#233](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md)

1. Would it make the template easier to read if you moved the [policy example](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md#1232-ensure-deployment-from-an-unauthorized-container-registry-is-denied) into the [intro section](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md#service-approval-accelerator-saa-template)?
2. When I looked at the template yesterday the benchmark link was broken, just checked and CIS site is down. I will check again later and add another comment. 
3. As a baseline would it be acceptable for a vendor to use their latest CIS benchmark e.g. Google using the GKE CIS Benchmark instead of the K8s benchmark
4. In the baseline would it be acceptable to just put a link to the CIS benchmark being used or are we expecting all the policies being included in the baseline to be documented in the service accelerator

eddie-knight commented:
Thanks @AdrianHammond! I added an answer to your question in the notes on the document, and adjusted the formatting to hopefully highlight those notes more clearly.

I didn't move the policy example yet (seemed good where it is), but we can discuss that more if you think it'll help.

Let me know what you think!

thinkl33t commented on templates/ServiceApprovalAcceleratorTemplate.md:
```diff
@@ -1,659 +1,70 @@
+_**Template Notes:**_
 
-**Controls and Architectures**
+- _Text here may be ommitted, or optionally used to provide any supplemental information related to the baseline._
```
omitted

eddie-knight replied:
corrected

thinkl33t commented on templates/ServiceApprovalAcceleratorTemplate.md:
```diff
@@ -1,659 +1,70 @@
-architectural best practices as documented in AWS’ public documentation, 
-white papers, and blog posts.
+The policies included in the following document(s) are required for compliance,
+unless otherwise noted in section 2: [Ammendments to the Baseline](#2-ammendments-to-the-baseline)
```
amendments

eddie-knight replied:
changed

abdullahgarcia commented:
Let's move things forward.

@AdrianHammond , this should help. Let us know if there are any questions.
